sklearn/tree/tests/test_tree.py

```diff
@@ -1831,7 +1874,7 @@ def test_empty_leaf_infinite_threshold():
         assert len(empty_leaf) == 0
 
 
-@pytest.mark.parametrize("criterion", CLF_CRITERIONS)
+@pytest.mark.parametrize("criterion", ["gini", "entropy", "hellinger"])
```

Comment:
```suggestion
@pytest.mark.parametrize("criterion", CLF_CRITERIONS + ["hellinger"])
```

Reply:
I don't see CLF_CRITERIONS definition in test_tree.py 
do you want me to add it ?
CLF_CRITERIONS  = ("gini", "entropy")

Reply:
It was defined previously, see: https://github.com/scikit-learn/scikit-learn/pull/16478/files#diff-e0c534f299457278700ee68add18acf0d4b2a1dd9391e1bcc49f3fe1b472fe67L65

Reply:
done